frontend/public/vendor.scss

```diff
@@ -57,7 +59,10 @@
 @import '~patternfly-react-extensions/dist/sass/filter-side-panel';
 @import '~patternfly-react-extensions/dist/sass/properties-side-panel';
 @import '~patternfly-react-extensions/dist/sass/vertical-tabs';
-@import '~@patternfly/patternfly/sass-utilities/all';
-@import "~@patternfly/patternfly/_variables";
+
+// Note: PF Sass load order is important and can cause issues if changed
 @import "~@patternfly/patternfly/_fonts";
 @import "~@patternfly/patternfly/_base";
+@import '~@patternfly/patternfly/layouts/Flex/flex';
```

Comment:
Do we need these extra imports if we aren't using the flex utility classes?

Reply:
i've removed the flex utils and spacing utils. I am using one display util in storage-class.tsx here:

`pf-u-display-none-on-md pf-u-display-inline-block-on-lg`

these display helpers are already responsive and MQ safe. If you want to remove them we can though, but I find them quite comparable to Bootstrap's `hidden-[breakpoint]` classes.